features/bevragen/zoek-fout-cases.feature

```diff
@@ -105,13 +105,13 @@ Rule: als contenttype voor de response wordt alleen application/json en charset
     | fields              | burgerservicenummer             |
     | header: Accept      |                                 |
     Dan heeft de response een object met de volgende gegevens
-    | naam     | waarde                                                                      |
-    | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.6                 |
-    | title    | Gevraagde contenttype wordt niet ondersteund.                               |
-    | detail   | Ondersteunde content type: */*, application/json. Default: application/json |
-    | code     | notAcceptable                                                               |
-    | status   | 406                                                                         |
-    | instance | /haalcentraal/api/brp/personen                                              |
+    | naam     | waarde                                                                                                                                                                              |
+    | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.6                                                                                                                         |
+    | title    | Gevraagde contenttype wordt niet ondersteund.                                                                                                                                       |
+    | detail   | Ondersteunde content type: */*, */*;charset=utf-8, */*; charset=utf-8, application/json, application/json; charset=utf-8, application/json;charset=utf-8. Default: application/json |
```

Comment:
is het noodzakelijk om '*/*;charset=utf-8' en '*/*; charset=utf-8' als ondersteunde content type op te nemen? Waarom dan ook niet */*; charset=UTF-8? En is dan '*/*;  charset=utf-8' dan niet meer valide? Maw is één (*/*; charset=utf-8) niet voldoende?

Reply:
@KayodeBakker technisch is het voor jou wellicht nodig om deze varianten te onderscheiden, maar voor een afnemer is dat niet zo relevant, toch? In feite is er maar 1 ondersteund formaat: application/json;charset=utf-8. Dat je dat op een beperkt aantal manieren mag opnemen (met of zonder spatie, kleine of grote letters, default weglaten) is niet zo relevant, toch?